lib/bugsnag/configuration.rb

```diff
@@ -201,6 +207,7 @@ def initialize
       self.send_environment = false
       self.send_code = true
       self.meta_data_filters = Set.new(DEFAULT_META_DATA_FILTERS)
+      @redacted_keys = Set.new
```

Comment:
... in fact maybe that is correct before `meta_data_filters` is removed to avoid a change in behaviour?

Then again, perhaps the longer list of defaults is correct for Ruby/Rails?

Reply:
We can't add defaults for redacted keys until `meta_data_filters` is removed, otherwise you'd have to remove the value from both to prevent it being redacted/filtered